<ui version="4.0" >
 <class>NewNetGameDialog</class>
 <widget class="QDialog" name="NewNetGameDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Netzwerkspiel erstellen</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="data/gui.qrc" >:/toolbar/filenew.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_use_pass" >
       <property name="text" >
        <string>&amp;Passwort:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="password" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Best√§tigung:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>password2</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="password2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Rundenbegrenzung:</string>
       </property>
       <property name="buddy" >
        <cstring>max_turns</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="max_turns" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_use_pass</tabstop>
  <tabstop>password</tabstop>
  <tabstop>password2</tabstop>
  <tabstop>max_turns</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="data/gui.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewNetGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>236</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewNetGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_use_pass</sender>
   <signal>toggled(bool)</signal>
   <receiver>password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>224</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_use_pass</sender>
   <signal>toggled(bool)</signal>
   <receiver>password2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>231</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
